src/libraries/System.Private.CoreLib/src/System/Collections/Generic/HashSetEqualityComparer.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Collections.Generic
+{
+    /// <summary>Equality comparer for hashsets of hashsets</summary>
```

Comment:
It may be worth fixing the comment: "Equality comparer for hash sets".  While it can be used for hash sets of hash sets, it does not have to. :)

Reply:
The only use of this is being returned from the CreateSetComparer method. On top of that, my care factor about this file is very low :-) I'm going to leave the comment.